CHANGELOG.md

```diff
@@ -17,6 +17,9 @@
 
 ## 2.0.97
 
+#### Changes
```

Comment:
Next release will be 2.0.98
Please update changelog and version to match

Reply:
noted